### ISSUE TITLE
Cannot write arrow functions that return object literals

### ISSUE THREAD

**Issue by kunalgolani** [babel/babel#1214]
without the arrow function transformer in use,

```js
	var foo = arr.map(v => ({
			x: v.bar,
			y: v.bar*2
		}));
```
transforms to

```js
	var foo = arr.map(v => {
			x: v.bar,
			y: v.bar*2
		});
```
removing the extra parentheses around the object literal and causing a parse error as the object literal is then seen as a function body.

to make it work, have to write source like this:
```js
	var foo = arr.map(v => {
			return {
				x: v.bar,
				y: v.bar*2
			}
		});
```

**Comment by kunalgolani** [comment]
Love the superfast turnaround!

**Comment by sebmck** [comment]
Sorry for the delay. Released as of 5.1.0, thanks!
